importScripts/refineryParameters/lowestLocation.ca_storage_locations.related.storageLocationSplitter.json

```diff
@@ -0,0 +1,100 @@
+{
+    "storageLocationType": "^lowestLocationType",
+    "relationshipType": "related",
+    "attributes": {
+        "idno": "",
+        "is_enabled": true,
+        "status": "completed"
+    },
+    "matchOn": [
+        "label"
+    ],
+    "parents": [
+        {
+            "name": "^Drawer",
```

Comment:
Now the source data has been split into Room, Drawer, Shelf and Box. Drawer and Shelf have same level of granularity. Boxes can occur underneath Room, Drawer or Shelf.